featurebyte/service/feature_table_cache.py

```diff
@@ -99,32 +100,25 @@ def __init__(
         self.tile_cache_service = tile_cache_service
         self.feature_list_service = feature_list_service
 
-    async def get_non_cached_nodes(
+    async def nodes_to_defition_hashes(
```

Comment:
Should the method name be `definition_hashes_to_nodes` if the return type is `Dict[str, Node]`?